### PR TITLE
Assignment 9: first attempt

### PR DESCRIPTION
**Pull request by enviro3**
-added thumb to the seek bar 
@droberts84

### CODE REVIEW

**Review by drobtravels — APPROVED** [review]
👍

File: app/scripts/directives/seekBar.js [hunk 1]
```diff
@@ -30,6 +30,10 @@
                     return {width: percentString()};
                 };
                 
+                scope.thumbStyle = function(){
+                    return {width: 10};
```
Comment (drobtravels) [inline app/scripts/directives/seekBar.js]: So this sets the thumbnail to always have a width 10 at all times.. but it stays in place.  Don't we want the thumb to move around as you drag your mouse?

**Review by drobtravels — CHANGES_REQUESTED** [review]
HINT: Checkout the CSS [left property](https://developer.mozilla.org/en-US/docs/Web/CSS/left)

File: app/scripts/directives/seekBar.js (hunk 1 above)
Comment (enviro3) [reply app/scripts/directives/seekBar.js]: so, i'm playing with it and used ```left: 10```. I don't know how to make the thumb nail move with the percentage bar

Comment (drobtravels) [reply app/scripts/directives/seekBar.js]: how do you make the percentage bar more?  Should left always be 10.. or will it need to change?

**Comment by drobtravels** [comment]
:+1:

**Pull request merged**